src/components/shared/editable-text-input-list/editable-text-input-list.tsx

```diff
@@ -3,37 +3,65 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { Button, Form, InputGroup } from "react-bootstrap";
+import { Button, Form, InputGroup } from 'react-bootstrap';
+import PageBlock from '../page-block/page-block';
 
 interface EditableTextInputListProps {
+  title: string;
   items: any[];
+  readOnly?: boolean;
+  ordered?: boolean;
   add: (val: any) => any;
   remove: (idx: number) => any;
   update: (idx: number, val: any) => any;
 }
 
 const EditableTextInputList: React.FC<EditableTextInputListProps> = ({
+  title,
   items,
+  readOnly,
+  ordered,
```

Comment:
I'm not sure how `ordered` would work in this type of list since the list items are text inputs and those can be changing at any time, so depending on what is in each text input at a given time, the order may be different. How were you envisioning `ordered` to work?

Reply:
I think I mentioned the whole list tag stuff should be removed

Reply:
Yeah, I saw that - semantically speaking, it would make sense to make this use a list tag as opposed to `div`s. Only caveat would be to remove all the default stylings on `li`, `ul`, and `ol` tags and then apply the styling that is currently on the `div`

Reply:
When I implemented this, I was looking at the previous implementation (the BulletList component) to see how I could port it over to the complete EditableTextInputList. Should I remove this or keep it in?

Reply:
If it's easier for you to keep it, I think it's fine to keep it. We can always make a ticket (and I hope we do) to refine EditableTextInputList in the future